src/node-file-trace.ts

```diff
@@ -79,6 +81,9 @@ export class Job {
     ts = true,
     analysis = {},
     cache,
+    // we use a default of 1024 concurrency to balance
+    // performance and memory usage for fs operations
+    fileIOConcurrency = Number(process.env.FILE_IO_CONCURRENCY) || 1024,
```

Comment:
Let not use an env var. Instead, the consumer should pass the `fileIOConcurrency` option to nodeFileTrace()

We also need to update the README.md with the new option